internal/server/audit/types_test.go

```diff
@@ -0,0 +1,150 @@
+package audit
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.flipt.io/flipt/rpc/flipt"
+)
+
+func TestFlag(t *testing.T) {
+	f := &flipt.Flag{
+		Key:          "flipt",
+		Name:         "flipt",
+		NamespaceKey: "flipt",
+		Enabled:      false,
+	}
+	nf := NewFlag(f)
+
+	assert.Equal(t, nf.Enabled, f.Enabled)
+	assert.Equal(t, nf.Key, f.Key)
+	assert.Equal(t, nf.Name, f.Name)
+	assert.Equal(t, nf.NamespaceKey, f.NamespaceKey)
+}
+
+func TestVariant(t *testing.T) {
+	v := &flipt.Variant{
+		Id:      "this-is-an-id",
+		FlagKey: "flipt",
+		Key:     "flipt",
+		Name:    "flipt",
+	}
+
+	nv := NewVariant(v)
+	assert.Equal(t, nv.Id, v.Id)
+	assert.Equal(t, nv.FlagKey, v.FlagKey)
+	assert.Equal(t, nv.Key, v.Key)
+	assert.Equal(t, nv.Name, v.Name)
+}
+
+func testConstraintHelper(t *testing.T, c *flipt.Constraint) {
```

Comment:
nit: could we add `t.Helper()` as first lines in all these helper methods? https://pkg.go.dev/testing#T.Helper

Reply:
@markphelps Cool will do that now!